### DISCUSSION TITLE
How to scroll the list

### DISCUSSION THREAD

**Discussion by gopusrikanth04** [xyflow/xyflow#2085] [Q&A] [answered]
Hello Team,
  
   I need this this of mapping fields functionality, Instead of zoom in and Zoom out I need scroll the lists separately
   
   How can I achieve this. Any Help  

<img width="928" alt="new mapping editor" src="https://user-images.githubusercontent.com/47174113/163754522-ff67cd51-7c6e-4c89-b750-407fd4b8d2c4.png">

**Comment by PoojaHoney** [comment]
Hello,

I too need this type of mapping functionality.

Any Help on this.

**Comment by bcakmakoglu** [comment, marked as answer]
@gopusrikanth04 @PoojaHoney 

I haven't tested it out but you can just add the `noWheelClassName` (which defaults to `nowheel` afaik) to your lists parent element and it would prevent zoom behavior while scrolling inside that element.

```jsx
<div className="nowheel">
	<!--  the rest of your jsx -->
</div>
```

You can also specifiy which class name to use by passing `noWheelClassName` to the ReactFlow component.

```jsx
<ReactFlow noWheelClassName="myNoWheelClassName" />
```
